clients/fides-js/src/lib/consent-types.ts

```diff
@@ -137,7 +137,9 @@ export interface FidesGlobal extends Fides {
   saved_consent: NoticeConsent;
   gtm: typeof gtm;
   identity: FidesJSIdentity;
+  prevConfig: FidesConfig | undefined;
   init: (config: FidesConfig) => Promise<void>;
+  reinit: () => Promise<void>;
```

Comment:
two nits:
1. Can we use `.reinitialize` instead? I did some polling of function names and I think that's the winner and I'm not in love with the overly-abbreviated `reinit` (mostly since `init()` isn't really expected to be used by a developer: https://ethyca.com/docs/dev-docs/js/reference/interfaces/Fides#init)
2. Can you update the `Fides` type imported from the `docs/` module to include this function and document there 👍

Reply:
Agreed, done and done.